soc/arm/silabs_exx32/efm32gg11b/dts_fixup.h

```diff
@@ -7,9 +7,6 @@
 
 /* SoC level DTS fixup file */
 
-#define DT_FLASH_DEV_BASE_ADDRESS	DT_SILABS_GECKO_FLASH_CONTROLLER_40000000_BASE_ADDRESS
-#define DT_FLASH_DEV_NAME		DT_SILABS_GECKO_FLASH_CONTROLLER_40000000_LABEL
-
 #define DT_RTC_0_NAME			DT_LABEL(DT_INST(0, silabs_gecko_rtcc))
```

Comment:
Maybe we could remove this line. Other gecko SoCs also support `counter_gecko_rtcc.c` driver but don't define `DT_RTC_0_NAME` so this is a bit inconsistent. We need to update samples / tests to use aliases or the chosen node anyway.

We can also clean it up later and merge this PR as is.

Reply:
Let's clean it up later.  There's a common issue w/ DT_RTC_0_NAME for other SoCs.  So when we have a solution for them we'll deal with this one silabs SoC as well.